src/WsSecurity.php

```diff
@@ -46,22 +61,38 @@ public function getSecurity()
     {
         return $this->security;
     }
+
     /**
      * Create the SoapHeader object to send as SoapHeader in the SOAP request.
+     *
      * @param string $username
      * @param string $password
-     * @param bool $passwordDigest
-     * @param int $addCreated
-     * @param int $addExpires
-     * @param bool $mustunderstand
+     * @param bool   $passwordDigest
+     * @param int    $addCreated
+     * @param int    $addExpires
+     * @param bool   $returnSoapHeader
+     * @param bool   $mustunderstand
      * @param string $actor
      * @param string $usernameId
-     * @param bool $addNonce
+     * @param bool   $addNonce
+     * @param string $envelopeNamespace
+     *
      * @return \SoapHeader|\SoapVar
      */
-    public static function createWsSecuritySoapHeader($username, $password, $passwordDigest = false, $addCreated = 0, $addExpires = 0, $returnSoapHeader = true, $mustunderstand = false, $actor = null, $usernameId = null, $addNonce = true)
-    {
-        $self = new WsSecurity($username, $password, $passwordDigest, $addCreated, $addExpires, $mustunderstand, $actor, $usernameId, $addNonce);
+    public static function createWsSecuritySoapHeader(
+        $username,
+        $password,
+        $passwordDigest = false,
+        $addCreated = 0,
+        $addExpires = 0,
+        $returnSoapHeader = true,
+        $mustunderstand = false,
+        $actor = null,
+        $usernameId = null,
+        $addNonce = true,
+        $envelopeNamespace = 'SOAP-ENV'
```

Comment:
Did you know I can be annoying :smile:?

```suggestion
        $envelopeNamespace = Security::ENV_NAMESPACE
```